### DISCUSSION TITLE
Dracula theme for Aseprite

### DISCUSSION THREAD

**Discussion by Abbabon** [dracula/dracula-theme#410] [General] [answered]
I've recently uploaded the Aseprite theme pack for Dracula:
https://github.com/dracula/aseprite-dracula-theme

What do you say, anything missing? Can we link to it from the landing page? :)

**Comment by Abbabon** [comment]
I will ping this in hope @zenorocha or one of his deputies sees it :)

**Comment by hacknug** [comment]
@Abbabon I seethe repo was already transferred to the organization. Care if I close this?

**Reply by benknoble** [reply]
Er, this is a 404: https://draculatheme.com/aseprite

**Comment by zenorocha** [comment, marked as answer]
Aaaaand we're live! https://draculatheme.com/aseprite

Thanks for your patience on this :D

Good job @Abbabon 🧛‍♂️

**Reply by Abbabon** [reply]
Noice :)
